samples/charts/financial-chart/axis-types/Pages/FinancialChartAxisTypes.razor

```diff
@@ -0,0 +1,61 @@
+﻿@page "/FinancialChartAxisTypes"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@using Samples.Services
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <label class="igOptions-label">X-Axis Mode: </label>
+        <label class="igOptions-item">
+            <select @bind="@XAxisMode">
+                <option>@FinancialChartXAxisMode.Time</option>
+                <option>@FinancialChartXAxisMode.Ordinal</option>
+            </select>
+            <label class="igOptions-label">Y-Axis Mode: </label>
+            <select @bind="@YAxisMode">
+                <option>@FinancialChartYAxisMode.PercentChange</option>
+                <option>@FinancialChartYAxisMode.Numeric</option>
```

Comment:
swap positions of above 2 lines because this sample is using Numeric as default YAxis mode

Reply:
pending